go.mod

```diff
@@ -74,4 +78,5 @@ replace (
 	// antrea/plugins/octant/go.mod also has this replacement since replace statement in dependencies
 	// were ignored. We need to change antrea/plugins/octant/go.mod if there is any change here.
 	github.com/contiv/ofnet => github.com/wenyingd/ofnet v0.0.0-20210318032909-171b6795a2da
+	github.com/googleapis/gnostic v0.5.5 => github.com/googleapis/gnostic v0.4.1
```

Comment:
typically we had a comment to explain why a replace directive is needed

Reply:
done.